src/Swashbuckle.AspNetCore.SwaggerGen/DependencyInjection/SwaggerGenOptionsExtensions.cs

```diff
@@ -556,6 +558,26 @@ public static void SupportNonNullableReferenceTypes(this SwaggerGenOptions swagg
             swaggerGenOptions.IncludeXmlComments(() => new XPathDocument(filePath), includeControllerXmlComments);
         }
 
+        /// <summary>
+        /// Inject human-friendly descriptions for Operations, Parameters and Schemas based on XML comments
+        /// from specific Assembly
+        /// </summary>
+        /// <param name="swaggerGenOptions"></param>
+        /// <param name="assembly">Assembly that contains XML Comments</param>
+        /// <param name="includeControllerXmlComments">
+        /// Flag to indicate if controller XML comments (i.e. summary) should be used to assign Tag descriptions.
+        /// Don't set this flag if you're customizing the default tag for operations via TagActionsBy.
+        /// </param>
+        public static void IncludeXmlCommentsForAssembly(
```

Comment:
I'm not sure we need the `ForAssembly` prefix, as isn't that implied by the parameters and /// docs already?

Reply:
Ok, removed it